### ISSUE TITLE
Progress Tracker

### ISSUE THREAD

**Issue by TurtleException** [TurtleException/TurtleCore#8]
A list of features that have to be implemented before the first release.

I didn't include most of the stuff that's already done. Otherwise this would be a lot longer c:
# Required goals
These things just have to be done before the first release or at least have some workaround:
- [x] #1 
- [ ] Use & document UTC standard
- [x] #3 
- [x] #4 (+ Testing)
- [ ] #5 & Protect against SQL-Injection
- [ ] #7 
- [ ] Complete documentation of `client.api` (Everything else is optional)
- [x] #11 
- [ ] #12 
- [ ] Create some sort of log
  - Ticket System
  - Changes on data / resources
  - Link / unlink actions of Users 
- [ ] #9 
- [ ] Introduce TODO list feature
# Optional goals
If some or all of these goals are not already done with the first release they might either be a good point to start after the release or may become long-term goals:
- [ ] Update wiki
- [ ] #2 
- [ ] Introduce partial objects instead of reference ids
- [ ] Try to avoid using Server `Main.singleton`
- [ ] Implement heartbeat
- [ ] Create "pronouns" field for User

After all of that is done: **Testing!**

**Comment by TurtleException** [comment]
Superseded by #16
